fe/fe-core/src/main/java/com/starrocks/catalog/DynamicPartitionProperty.java

```diff
@@ -84,6 +89,10 @@ public DynamicPartitionProperty(Map<String, String> properties) {
             this.buckets = Integer.parseInt(properties.get(BUCKETS));
             this.replicationNum =
                     Integer.parseInt(properties.getOrDefault(REPLICATION_NUM, String.valueOf(NOT_SET_REPLICATION_NUM)));
+            this.createHistoryPartition =
+                    Boolean.parseBoolean(properties.get(CREATE_HISTORY_PARTITION));
```

Comment:
CREATE_HISTORY_PARTITION seems unnecessary because the user setting HISTORY_PARTITION_NUM means that he wants to create history partitions.

Reply:
you are right.